packages/types/src/providers/bedrock.ts

```diff
@@ -68,6 +68,21 @@ export const bedrockModels = {
 		maxCachePoints: 1,
 		cachableFields: ["system"],
 	},
+	"amazon.nova-2-lite-v1:0": {
+		maxTokens: 5000,
```

Comment:
This seems low - does it work to match what openrouter does?
```suggestion
		maxTokens: 65_536,
```